### DISCUSSION TITLE
can't find file in dataset.

### DISCUSSION THREAD

**Discussion by abhinav9898** [Guvi-MentorDE/BE_batch_Guvi_Aug6#50] [General]
Worksheet 3 uses a raw dataset "dept_loc.txt" which has to be loaded into hive database, not able to see that file in hadoop>datasets. 
Am i missing something?

**Comment by Guvi-MentorDE** [comment]
hi @abhinav9898  , please check the hive folder now I have uploaded the missing datasets.

**Reply by abhinav9898** [reply]
Thanks @Guvi-MentorDE
